web/libs/editor/src/components/CurrentEntity/Controls.jsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * @deprecated This component is not used anywhere.
```

Comment:
this is confusing, github is reporting a new file but its deprcated? feels like a file rename. is that what happened or am I missing something?

Reply:
i just accidentally re-intoduced some files that were removed in upstream. removed